rnd/autogpt_server/autogpt_server/cli.py

```diff
@@ -2,86 +2,94 @@
 The command line interface for the agent server
 """
 
-from multiprocessing import freeze_support
-from multiprocessing.spawn import freeze_support as freeze_support_spawn
+import os
+import pathlib
 
 import click
+import psutil
 
+from autogpt_server import app
+from autogpt_server.util.process import AppProcess
 
-@click.group()
-def main():
-    """AutoGPT Server CLI Tool"""
 
+def get_pid_path() -> pathlib.Path:
```

Comment:
Change here mostly enforcing the use of psutils instead of UNIX function to allow multiplatform.
Also extracting the process creation logic out of this file.